### ISSUE TITLE
Command line tools to create DAS server

### ISSUE THREAD

**Issue by andre-senna** [singnet/das-query-engine#107]
Create an easy-to-use set of command line tools to setup a DAS server with a knowledge base. These tools should include:

* Setup of a custom DAS server in localhost
* Setup of a openfaas deployment in localhost
* Pre-check of MeTTa knowledge base files syntax
* Documentation of all tools

**Comment by levisingularity** [comment]
@andre-senna This is the first item of this issue. What do you think?

https://github.com/singnet/das-query-engine/assets/141682181/a223ece6-9682-4daf-885b-eacc2b361e86

I've noticed some print statements in a few commands that I plan to remove before releasing this feature. I just want to confirm if I'm aligning with your vision in this first topic of the feature.

**Comment by andre-senna** [comment]
@levisingularity 

Yes I think this is well aligned with our recent discussions. I have just a couple of minor comments you may want to consider:

(1) The value presented as default (in the case of the user just hitting <enter> without typing anyting) you present between [] should be taken from the config file you use as output. If there's no such file, only in this case you present a default which is set in your script. This way if the user can `server configure` for the second time, all the defaults would be the values already entered before.

(2) I'm not sure this is suitable with the way you designed the script, if not we can discuss a better idea but I would have two commands `server config list` and `server config set`. The first one just print current configuration and the second one do as you already did.

(3) `server start` should print a message in stdout like "DAS server listening on port X" or something alike just to make the user aware of the port being used.
